the-events-calendar.php

```diff
@@ -27,11 +27,10 @@
 Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 
+define( 'THE_EVENTS_CALENDAR_DIR', dirname( __FILE__ ) );
```

Comment:
I would avoid two variables. Just the file would do. You can Create the other variable on the Class no need to store one more constant.